### PR TITLE
feat(EMS-2197): Accessibility statement update

### PR DESCRIPTION
**Pull request by Zainzzkk**
## Introduction ✏️ 
This PR updates the accessiblity statement for both quote and insurance tools

## Resolution ✔️ 
- Updated accessibility statement content strings
- Updated template and changed order
- Updated e2e tests

### CODE REVIEW

File: e2e-tests/content-strings/pages/index.js [hunk 1]
```diff
@@ -45,11 +45,8 @@ const CANNOT_APPLY = {
 };
 
 const ACCESSIBILITY_STATEMENT_PAGE = {
-  PAGE_TITLE: 'Accessibility Statement for the EXIP Get a Quote Service',
-  SERVICE_LINK: {
-    TEXT: 'Get a quote for UKEF export insurance - GOV.UK (www.gov.uk)',
-    HREF: LINKS.EXTERNAL.BEFORE_YOU_START,
-  },
```
Comment (ttbarnes) [inline e2e-tests/content-strings/pages/index.js:52]: service link removed?

Comment (Zainzzkk) [reply e2e-tests/content-strings/pages/index.js:52]: its not on the word document

File: e2e-tests/insurance/cypress/e2e/journeys/accessibility-statement.spec.js [hunk 2]
```diff
@@ -114,6 +105,44 @@ context('Accessibility statement page - Insurance', () => {
...
+    it('renders a list', () => {
+      complianceStatus.listItem1().invoke('text').then((text) => {
+        expect(text.trim()).equal(`${COMPLIANCE_STATUS.LIST[0]} ${COMPLIANCE_STATUS.LIST[1]}`);
+      });
+
+      complianceStatus.listItem2().invoke('text').then((text) => {
+        expect(text.trim()).equal(`${COMPLIANCE_STATUS.LIST[2]} ${COMPLIANCE_STATUS.LIST[3]}`);
+      });
+    });
+  });
```
Comment (ttbarnes) [inline e2e-tests/insurance/cypress/e2e/journeys/accessibility-statement.spec.js:144]: `cy.checkText()`?

File: e2e-tests/insurance/cypress/e2e/journeys/accessibility-statement.spec.js [hunk 3]
```diff
@@ -56,14 +55,6 @@ context('Accessibility statement page - Insurance', () => {
     });
   });
 
-  it('renders a service link', () => {
-    cy.checkLink(
-      accessibilityStatementPage.serviceLink(),
```
Comment (ttbarnes) [inline e2e-tests/insurance/cypress/e2e/journeys/accessibility-statement.spec.js:61]: Can we remove the `serviceLink` selector if its no longer requireD?

File: e2e-tests/quote/cypress/e2e/journeys/quote/accessibility-statement.spec.js [hunk 4]
```diff
@@ -109,6 +100,44 @@ context('Accessibility statement page - Quote', () => {
...
+      complianceStatus.listItem1().invoke('text').then((text) => {
+        expect(text.trim()).equal(`${COMPLIANCE_STATUS.LIST[0]} ${COMPLIANCE_STATUS.LIST[1]}`);
+      });
+
+      complianceStatus.listItem2().invoke('text').then((text) => {
+        expect(text.trim()).equal(`${COMPLIANCE_STATUS.LIST[2]} ${COMPLIANCE_STATUS.LIST[3]}`);
+      });
+    });
+  });
+
```
Comment (ttbarnes) [inline e2e-tests/quote/cypress/e2e/journeys/quote/accessibility-statement.spec.js:140]: `cy.checkText()`?

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**